# Define a message type for passing an array of points to the server and getting the merged points back

# Define the action goal message
string client_id
Point2D[] points
---
# Define the action result message
Point2D[] merged_points
float32 radius_used
---
# Define the action feedback message
float32 percent_completed
